manifests/r/RedHat/Podman/3.3.1/RedHat.Podman.installer.yaml

```diff
@@ -0,0 +1,19 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: RedHat.Podman
+PackageVersion: 3.3.1
+MinimumOSVersion: 10.0.0.0
+Commands:
+- podman
+Installers:
+- Architecture: x64
```

Comment:
```suggestion
- Architecture: x86
```

Reply:
It installs fine on x86, but running `C:\Program Files\RedHat\Podman\podman.exe` doesn't work. Probably because the `podman.exe` is only compiled for x64.
![image](https://user-images.githubusercontent.com/11600822/134919006-7e762af3-f0ba-40ea-8d5c-dd3ea31c8bfe.png)
![image](https://user-images.githubusercontent.com/11600822/134919207-cbc67ac9-52e5-4e54-b253-9596da147bf2.png)

This is what happens on x64:
![image](https://user-images.githubusercontent.com/11600822/134920565-c13b0461-e90b-46a6-a9f1-506c6e3a22c1.png)
![image](https://user-images.githubusercontent.com/11600822/134920502-9248b592-dca4-4bfe-b9b6-3f7fe8b7db94.png)
![image](https://user-images.githubusercontent.com/11600822/134920613-5284c571-cf46-40c6-b87d-481dce3fa2b0.png)

Reply:
I fixed it, let's see if they merge it: https://github.com/containers/podman/pull/11759

Reply:
It’s been merged. I don’t know what their release schedule is though.